__tests__/app.test.js

```diff
@@ -41,4 +41,42 @@ describe('GET /api', () => {
             expect(body).toEqual(endpoints)
         })
     })
+})
+
+describe.only('GET /api/articles/:articleId', () => {
```

Comment:
Please get rid of the `.only` before committing. This will stop your other tests from running and these are needed for husky